src/styles/reset.css

```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
```

Comment:
I think this rule is redundant, because you have different sizes for font. You duplicate this rule in different classe - card__description, for example. But it's my suggestion only.

Reply:
yes, a good remark, i have removed font-size from html, every block has it own font-size